repo/packages/M/marathon/300/resource.json

```diff
@@ -0,0 +1,14 @@
+{
+  "images" : {
+    "icon-small" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-small.png",
+    "icon-medium" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-medium.png",
+    "icon-large" : "https://downloads.mesosphere.com/marathon/assets/icon-service-marathon-large.png"
+  },
+  "assets" : {
+    "container" : {
+      "docker" : {
+        "image" : "mesosphere/marathon:v1.7.174"
+      }
+    }
+  }
+}
```

Comment:
(super nit, feel free to ignore) newline missing (from all other files too).

Reply:
this files are generated... I will look to see if we can resolve it there.